frontend/src/app/shared/components/modals/preview-modal/hover-card-modal/hover-card.modal.sass

```diff
@@ -4,7 +4,8 @@
   position: absolute
   background-color: var(--body-background)
   z-index: 5000
-  min-width: 350px
-  box-shadow: var(--shadow-floating-large)
+  min-width: 500px
+  box-shadow: var(--shadow-floating-medium)
   pointer-events: all
-  padding: 1rem
+  padding: 0.75rem
```

Comment:
Instead of 0.75rem, we can use --overlay-paddingBlock-normal Primer variable. What's your opinion?